www/settings.py

```diff
@@ -0,0 +1,135 @@
+"""
+Django settings for www project.
+
+Generated by 'django-admin startproject' using Django 4.0.4.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.0/ref/settings/
+"""
+
+from pathlib import Path
+import os
+
+import dotenv
+
+dotenv.load_dotenv()
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
+
+SECRET_KEY = os.environ["DJANGO_SECRET_KEY"]
```

Comment:
Put some defaults in here. We'll override them in prod for sure, but it'll help start it up locally for easy local development.